bundle/regal/ast/keywords.rego

```diff
@@ -0,0 +1,132 @@
+package regal.ast
+
+import rego.v1
+
+keywords[row] contains keyword if {
+	some rule in input.rules
+
+	rule_text := base64.decode(rule.location.text)
+
+	some idx, line in split(rule_text, "\n")
```

Comment:
You've got the decoded text representation available in `input.regal.file.lines`... I think we could save some processing here (and in more places in this file) using that.

Reply:
This took me a while to test, but I have updated to do this in 73370ea after finding that it was in fact a little faster. The commit message has the testing details if interested.